src/modules/styling.js

```diff
@@ -0,0 +1,45 @@
+import { arrTasks } from './data.js';
+import { resetTasks } from './reset.js';
+import { addIconEvent, addInputDescEvent } from './task-edit.js'; // eslint-disable-line
+import checkChanged from './toggle.js';
+
+const compileTaskItem = (task) => {
+  const liTask = document.createElement('li');
+  liTask.id = `task-${task.index}`;
+  liTask.className = 'tasks';
+
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  checkbox.checked = task.completed;
+  checkbox.id = task.index;
+
+  const inputDesc = document.createElement('input');
+  inputDesc.id = `desc-${task.index}`;
+  inputDesc.type = 'text';
+  inputDesc.value = task.description;
+  inputDesc.className = 'task-desc';
+
+  const iCon = document.createElement('i');
+  if (checkbox.checked) {
+    iCon.classList.add('item-btns', 'bi', 'bi-trash3');
+    inputDesc.classList.add('completed');
+  } else {
+    iCon.classList.add('item-btns', 'bi', 'bi-three-dots-vertical');
+  }
+  iCon.id = `icon-${task.index}`;
+  liTask.append(checkbox, inputDesc, iCon);
+  return liTask;
+};
+
+const populateTaskList = () => {
```

Comment:
I see that you used another function to populate the list in `reset.js`. Your code would be more DRY if the function is defined in one file and called it to the other.